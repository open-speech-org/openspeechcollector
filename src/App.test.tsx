import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import configureStore, {history} from "./store/configureStore";
import {Provider} from "react-redux";
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  const store = configureStore();
  ReactDOM.render(
      <Provider store={store}>
      <App history={history} />
      </Provider>,
      div
  );
  ReactDOM.unmountComponentAtNode(div);
});


it('shallow app component snapshot', () => {
    const component = shallow(<App history={history} />);
    expect(component).toMatchSnapshot();
});