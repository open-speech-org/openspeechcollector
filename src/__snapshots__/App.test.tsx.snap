// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow app component snapshot 1`] = `ShallowWrapper {}`;
